### PR TITLE
mailgun plugin

### PR DESCRIPTION
**Pull request by lukvmil**
Finished and test mailgun plugin, ready to merge

### CODE REVIEW

**Comment by mashton** [comment]
Looks great! 🙌 

> django.db.migrations.exceptions.InvalidBasesError: Cannot resolve bases for [<ModelState: 'mailgun.Tutorial'>]
This can happen if you are inheriting models from an app with migrations (e.g. contrib.auth)
 in an app with no migrations; see https://docs.djangoproject.com/en/3.2/topics/migrations/#dependencies for more
Error: Process completed with exit code 1.

You need to run `python manage.py makemigrations mailgun` and commit the migration file that is generated. That should fix the build.

File: metagov/metagov/plugins/mailgun/models.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import metagov.core.plugin_decorators as Registry
+from metagov.core.models import Plugin
+
+import requests
+
+@Registry.plugin
+class Tutorial(Plugin):
```
Comment (mashton) [inline metagov/metagov/plugins/mailgun/models.py:7]: Rename `Tutorial`->`Mailgun`

Comment (lukvmil) [reply metagov/metagov/plugins/mailgun/models.py:7]: Ok great! I'll make those changes

**Comment by lukvmil** [comment]
Made changes and this branch now passes all checks, will merge now!

**Pull request merged**